contracts/mocks/SedaPermissioned.sol

```diff
@@ -135,6 +159,12 @@ contract SedaPermissioned is ISedaCore, RequestHandlerBase, AccessControl, Pausa
         return queriedPendingRequests;
     }
 
+    /// @inheritdoc ISedaCore
+    /// @dev This is a mock implementation that does nothing and requires admin role
```

Comment:
It doesn't require admin right?

Reply:
Tbh, I was tempted to remove the SedaPermissioned contract

Reply:
Yeah I missed that it was the permissioned/mock version when I made the comment :p